tests/libvmi/factory.go

```diff
@@ -89,3 +89,16 @@ func NewCirros(opts ...Option) *kvirtv1.VirtualMachineInstance {
 	cirrosOpts = append(cirrosOpts, opts...)
 	return New(RandName(DefaultVmiName), cirrosOpts...)
 }
+
+// NewAlpine instantiates a new Alpine based VMI configuration
+func NewAlpine(opts ...Option) *kvirtv1.VirtualMachineInstance {
+	alpineOpts := []Option{
+		WithContainerImage(cd.ContainerDiskFor(cd.ContainerDiskAlpine)),
+		WithCloudInitNoCloudUserData("#!/bin/bash\necho 'hello'\n", true),
```

Comment:
@EdDev please remind me why this is needed ....

I have a slight recollection that VMs spawned from certain images require cloud init for *something* to happen. Can't remember what the something was though ...

Reply:
Per what I remember, it defines an account so later one can login to its terminal.

Reply:
Is it for all image types, or just a subset of them ?

Reply:
I think we only adjusted the fedora image not to require this "hack", but if my memory is not failing me, @oshoval worked on it and may have more info.

Reply:
for fedora (only indeed) we removed the need to have cloud-init / cloud-init that sets the password (password comes already with the image).

Another thing that is caused by setting `WithCloudInitNoCloudUserData` is that kubevirt sets the localhostname in the cloudInit metadata internally

Here it is Alpine, and not for the password,
we still have specific tests the use the cloudInit for several reasons (some are just because it is a leftover)
i don't know specifically if here it is needed